src/explore-education-statistics-admin/src/services/userService.ts

```diff
@@ -43,6 +43,7 @@ export interface UserPublicationRoleSubmission {
 export interface UserInvite {
   email: string;
   roleId: string;
+  userReleaseRoles: { releaseId: string; releaseRole: string }[];
```

Comment:
Couldn't this just be `releaseRoles`?

Reply:
See previous comment: using `UserReleaseRole` to indicate it's a releaseId AND a releaseRole, not just a releaseRole.